### ISSUE TITLE
Typing open parenthesis is causing intellisense to auto complete with first result in candidate list

### ISSUE THREAD

**Issue by jaholme** [Microsoft/vscode#70070]
Issue Type: <b>Bug</b>

Have some sort of intellisense enabled on your c++ project.
Then type something like GetChars which will pull up a candidiate list with things like GetCharactersistics(), as soon as I type (, it auto selects the top item in the candidate list.  This used to not auto complete before.  

VS Code version: Code 1.32.1 (05f146c7a8f7f78e80261aa3b2a2e642586f9eb3, 2019-03-08T00:51:49.278Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700 CPU @ 3.40GHz (8 x 3408)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|31.79GB (18.98GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (10)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-tidyhtml|anw|1.10.0
xml|Dot|2.4.0
gitlens|eam|9.5.1
vscode-power-mode|hoo|2.2.0
vscode-idl|kur|0.1.2
vscode-nmake-tools|Mic|1.1.190220004
cpptools|ms-|0.21.0
csharp|ms-|1.17.1
PowerShell|ms-|1.11.0
team|ms-|1.149.2

</details>

**Comment by jrieken** [comment]
Please share you IntelliSense settings, e.g. setting starting/containing `suggest`

**Comment by jrieken** [comment]
@sean-mcmanus does the C++ extension use auto accept characters for completions? E.g. is `(` an accept character and did this chance recently?

**Comment by sean-mcmanus** [comment]
Yeah, we added that in 0.21.0, but our latest 0.22.0-insiders2 has that removed -- it was a 1 line change to add the '(' commit characters, but we later realized that it causes undesirable completions in certain cases (e.g. when typing a new function declaration/definition) so we removed it. We have an internal branch that uses snippets to auto add the `(|)` for function completions, but we're not shipping that until we add a setting to allow users to disable it.

**Comment by jrieken** [comment]
Thanks for clarifying
